.github/actions/docker/action.yml

```diff
@@ -63,13 +63,9 @@ inputs:
     default: 'false'
     description: 'Generate tag only.'
     required: false
```

Comment:
This input should be removed as well if we're eliminating `TAG_ONLY` entirely.

Reply:
i can keep it if we think it is or should be used? would need to understand the use case that led to it being added

Reply:
From what I can see, the only use I've seen so far is in the cd-retag workflow, which tags an existing image with another tag and pushes it. Let's keep things as-is for now.